hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -958,6 +965,26 @@ private void waitForHawkularServer() throws Exception {
     }
 
     /**
+     * If the server returns a 200 OK, we still need to check the content if the server
+     * is really up. This is explained here: https://twitter.com/heiglandreas/status/801137903149654017
+     * @param body_string String representation of the body
+     * @return true if it is really up, false otherwise (still starting).
+     */
+    private boolean check_really_up(String body_string) {
```

Comment:
You've been programming in Ruby too long. :) Standards for method names are camel case - please rename to checkReallyUp() and "body_string" to "bodyString". Same on line 914 (body_string == bodyString)